lib/shared/addon/components/container-shell/component.js

```diff
@@ -90,7 +90,14 @@ export default Component.extend(ThrottledResize, {
 
       url = `${ scheme }${ window.location.host }/k8s/clusters/${ clusterId }/api/v1/namespaces/${ namespaceId }/pods/${ podName }/exec`;
       url += `?container=${ encodeURIComponent(containerName) }&stdout=1&stdin=1&stderr=1&tty=1`;
-      const command = get(instance, 'command') || DEFAULT_COMMAND;
+      let command = get(instance, 'command') || DEFAULT_COMMAND;
+
+      const { node } = instance || {}
+      const system = node.info && node.info.os && node.info.os.operatingSystem || ''
```

Comment:
What are you trying to accomplish here?

Reply:
If a pod is deployed in windows node, 'Execute Shell' need `cmd` as the command to enter in. So I obtain the node os info and modify the command. ( Linux is `bash` and windows is `cmd`)